test/e2e/tests/transaction/send-eth.spec.js

```diff
@@ -201,6 +208,7 @@ describe('Send ETH', function () {
         {
           fixtures: new FixtureBuilder().build(),
           title: this.test.fullTitle(),
+          localNode: 'anvil',
```

Comment:
these specs are using anvil. In the subsequent PR, we'll make that localNode has 'anvil' by default, so we don't need to pass this option in all specs, rather we change the current default from 'ganache' to 'anvil'